cfme/tests/infrastructure/test_vmware_provider.py

```diff
@@ -278,3 +278,36 @@ def test_vmware_provisioned_vm_host_relationship():
             3.CFME Provisioned VM should have host relationship.
     """
     pass
+
+
+@pytest.mark.manual
+@pytest.mark.tier(1)
+def test_esxi_reboot_not_orphan_vms():
+    """
+    By mimicking ESXi reboot effect on VMs in CFME, make sure they are not getting marked orphaned.
+
+    Bugzilla:
+        1695008
+
+    Polarion:
+        assignee: kkulkarn
+        casecomponent: Infra
+        caseimportance: critical
+        initialEstimate: 1/2h
+        testtype: integration
+        testSteps:
+            1.Add VMware provider to CFME
+            2.SSH to CFME appliance and perform following steps in rails console
+                '''
```

Comment:
Q: why do you need these quotes here?

Reply:
I talked to @jawatts yesterday and he said that was okay. These are rails commands and I wanted to make sure they are treated as a block within this step. Justin do you think its ok to keep it ?

Reply:
@kedark3 I think it is okay to keep it.

One question after reading this PR, couldn't this be automated fairly quickly?